src/states/Lobby.gd

```diff
@@ -188,13 +188,11 @@ func update_table() -> void:
 
 # Update the enabled/disabled state for all buttons.
 func update_buttons() -> void:
-	for idx in range(len(COLORS)):
-		button_circle.get_node(str(idx)).disabled = (idx in chosen_colors.values())
-	if not get_tree().is_network_server():
-		start_button.disabled = true
-	else:
-		var all_players_selected := true
-		for player_id in Multiplayer.player_info.keys():
-			if not (player_id in chosen_colors):
-				all_players_selected = false
-		start_button.disabled = not (all_players_selected and len(chosen_colors.keys()) > 1)
+	if not Multiplayer.dedicated_server:
+		for idx in range(len(COLORS)):
+			button_circle.get_node(str(idx)).disabled = (idx in chosen_colors.values())
+	var all_players_selected := true
+	for player_id in Multiplayer.player_info.keys():
+		if not (player_id in chosen_colors):
+			all_players_selected = false
+	start_button.disabled = not (all_players_selected and len(chosen_colors.keys()) > 0)
```

Comment:
Yeah this function is gonna be a huge pain to fix conflicts for lol

Reply:
😩